freqtrade/freqai/base_models/BasePyTorchRegressor.py

```diff
@@ -47,4 +47,5 @@ def predict(
         y = self.model.model(x)
         y = y.cpu()
         pred_df = DataFrame(y.detach().numpy(), columns=[dk.label_list[0]])
+        pred_df = dk.denormalize_labels_from_metadata(pred_df)
```

Comment:
> That means that the denormalization is now properly handled in develop.

@robcaulk does this mean this line should be reverted, right?

Reply:
Yeah it’s a just a resolution on the merge conflict. They should look the same so just keep incoming.